CONTRIBUTING.md

```diff
@@ -139,8 +139,7 @@ your feature works as intended.
 
 After this point, you may be asked to make modification to your code to adhere
 to coherent code style or to fix bugs you may have not noticed. Once you get
-approvals from 1 or 2 repository owners, we will merge your code!
-:confetti_ball: :tada:
+approvals from 1 or 2 repository owners, we will merge your code! :confetti_ball: :tada:
 
 
 ## Releasing
```

Comment:
i think we can remove this entire section as it's not relevant for contributors and we have [this wiki page](https://github.com/palantir/blueprint/wiki/Release-process)